client/extensions/woocommerce/app/products/product-reviews-widget.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Would this be useful as a shared component, maybe for the dashboard?

Reply:
We don't have a use for it yet, but I could see it being useful in the future. It's already written in a way that it can be shared, so I've moved it to our `components/` folder.